### DISCUSSION TITLE
jq remove empty objects from objects array

### DISCUSSION THREAD

**Discussion by lamyergeier** [stedolan/jq#3169] [Q&A] [answered]
The following command removes empty entries from jq: `jq 'del(..|nulls)`
But additionally I would also like to remove empty objects in an object array, how to do it?

# Sample

```json
[
	{
		"a": "1"
	},
        {},
	{
		"a": "2"
	},
	{}
]
```

# Output desired

```json
[
	{
		"a": "1"
	},
	{
		"a": "2"
	}
]
```

**Comment by emanuele6** [comment, marked as answer]
`del(.[] | select(. == {}))`

**Comment by pkoppstein** [comment]
@lamyergeier - Many usage questions about jq already have answers at the [jq FAQ](https://github.com/jqlang/jq/wiki/FAQ) or [stackoverflow.com](https://stackoverflow.com/questions/tagged/jq).  

In future, if you find your question about usage does not have a satisfactory answer at one of these locations, please post your question at stackoverflow.com using the jq tag.

**Reply by emanuele6** [reply]
I don't have a stackoverflow account.
